aviary/interface/methods_for_level1.py

```diff
@@ -130,32 +121,7 @@ def run_level_1(
     # else:
     kwargs['optimizer'] = optimizer
 
-    if phase_info is None:
-        if mission_origin == 'GASP':
-            from aviary.interface.default_phase_info.gasp import phase_info, phase_info_parameterization
-            kwargs['phase_info_parameterization'] = phase_info_parameterization
-
-        else:
-            from aviary.interface.default_phase_info.flops import phase_info, phase_info_parameterization
-            kwargs['phase_info_parameterization'] = phase_info_parameterization
-    else:
-        # Load the phase info dynamically from the current working directory
-        phase_info_module_path = os.path.join(os.getcwd(), "outputted_phase_info.py")
-        if not os.path.exists(phase_info_module_path):
-            raise FileNotFoundError(
-                "The outputted_phase_info.py file is not in the current working directory. Please run `draw_mission` to generate this file.")
-
-        spec = importlib.util.spec_from_file_location(
-            "outputted_phase_info", phase_info_module_path)
-        outputted_phase_info = importlib.util.module_from_spec(spec)
-        sys.modules["outputted_phase_info"] = outputted_phase_info
-        spec.loader.exec_module(outputted_phase_info)
-
-        # Access the phase_info variable from the loaded module
-        phase_info = outputted_phase_info.phase_info
-
-    prob = run_aviary(input_deck, phase_info, mission_method=mission_origin,
-                      mass_method=mass_origin, **kwargs)
+    prob = run_aviary(input_deck, phase_info, **kwargs)
```

Comment:
This logic got moved to methods_for_level2.py - since the AviaryProblem is now getting EOM type from reading the input deck, everything related to logic branches based on EOM have to get moved down the chain.